src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carCostDay = 40;
+  const rentDays = days;
+  const basicDiscount = 50;
+  const minDiscount = 20;
+  let total = carCostDay * rentDays;
+
+  if (rentDays > 2 && rentDays <= 6) {
+    total -= minDiscount;
+  }
+
+  if (rentDays > 6) {
+    total -= basicDiscount;
+  }
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 3rd item, you can return values from if immediately